projects/sbb-esta/angular-business/src/lib/header/header/header.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="sbb-header-ribbon" *ngIf="environment" [style.background]="environmentColor">
+  {{ environment }}
+</div>
+
+<div class="sbb-header-appchooser">
+  <span class="sbb-svgsprite-icon">
+    <!-- placeholder for other icon -->
+    <sbb-icon-hamburger-menu></sbb-icon-hamburger-menu>
+  </span>
+  <!-- Content here -->
+</div>
+
+<div class="sbb-header-titlebox">
+  <div class="sbb-header-titlebox-label">{{ label }}</div>
+  <div *ngIf="subtitle" class="sbb-header-titlebox-subtitle">{{ subtitle }}</div>
+</div>
+
+<div class="sbb-header-mainnavigation">
+  <template #navigationButtons></template>
+</div>
+<div class="sbb-header-usermenu">
+  <ng-content select="sbb-usermenu"></ng-content>
+</div>
+
+<!-- If an icon is provided, use that, otherwise use the sbb logo -->
+<!-- TODO: make cleaner -->
+<div #iconContent class="sbb-header-logo">
+  <ng-content select="[brand],.brand"></ng-content>
+</div>
+<div *ngIf="!isIconProvided" class="sbb-header-logo">
```

Comment:
You can replace !isIconProvided with !iconContent.childElementCount  and remove the iconContent property in the .ts file

Reply:
I had tried that in the beginning and had some issues with undefined values, but probably it was due to side effects which I should've solved by now.
Regardless, I'll remove the property.